internal/docker/client.go

```diff
@@ -26,12 +27,24 @@ import (
 	"github.com/moby/buildkit/session/auth/authprovider"
 	"github.com/opentracing/opentracing-go"
 	"github.com/pkg/errors"
+	"github.com/windmilleng/tilt/internal/sliceutils"
 
 	"github.com/windmilleng/tilt/internal/container"
 	"github.com/windmilleng/tilt/pkg/logger"
 	"github.com/windmilleng/tilt/pkg/model"
 )
 
+// Label that we attach to all of the images we build.
+const (
+	BuiltBy = "builtby"
+	Tilt    = "tilt"
```

Comment:
I think `client.Tilt` is a weird name for a constant that is just "tilt". Is this referenced anywhere else? Does it need to be a constant?